scripts/jtsync/README.md

```diff
@@ -0,0 +1,58 @@
+# jtsync - synchronize status from jenkins to trello
+    Usage: jtsync --ci SERVICE (--matrix|--job) JOB_STATUS
+            --ci SERVICE                 Which ci is used (suse or opensuse)
+            --matrix NAME,PROJECT        Set status of a matrix job
+            --job NAME                   Set status of a normal job
+
+Matrix job cards are selected in this form:
+
+    $ jtsync --ci suse --matrix crowbar-trackupstream,Devel:Cloud:7:Staging
```

Comment:
The job is called cloud-trackupstream. But this doesn't really work, does it? When there is a series in the mateix job under that project and one in the middle fails, the only thing that this will do is flip the label.